package.json

```diff
@@ -29,8 +29,8 @@
   "scripts": {
     "postinstall": "if [ $SKIP_POSTINSTALL != \"true\" ]; then npm run setup-hooks; else echo \"Post install script skipped as the env SKIP_POSTINSTALL is set to true.\"; fi",
     "setup-hooks": "git config core.hooksPath ./scripts/git-hooks",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "react-scripts --max_old_space_size=4096 start",
+    "build": "react-scripts --max_old_space_size=4096 build",
```

Comment:
Theoretically we could try to avoid such flag by disabling sourcemap generation, but I believe it may be useful for us, especially while we are at so early stage with this UI.

See:
- https://github.com/facebook/create-react-app/issues/8413#issuecomment-924223670
- https://create-react-app.dev/docs/advanced-configuration/